Kernel/Prekernel/StdLib.cpp

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Prekernel/StdLib.h>
+
+void halt()
```

Comment:
StdLib.cpp seems like the wrong file name for a halt() function IMO,
but I can't think of a good name either